wiki/Beatmap_Discussion/en.md

```diff
@@ -1,225 +1,183 @@
+- Pending: Shows pending posts
+- Praises: Shows praise and hype posts
+- All: Shows all posts
 
-- Mine: Mods you have contributed.
-- Resolved: Mods the creator has responded to and marked as resolved.
-- Pending: Mods that haven't been replied to or marked as resolved.
-- Praises: Mods that have been submitted as a praise
-- All: Every suggestion, problem, and praise posted on the beatmap's difficulty and the beatmapset's general section.
+## Visual timeline
 
-### Modding timeline
+![](img/visual-timeline.jpg "Mod posts spread out on the timeline.")
 
-![](img/timeline.jpg "Mod posts, time-stamped")
+The visual timeline displays all of the mods made for a difficulty which use a timestamp. Clicking on any of these will scroll down to the mod posted at that timestamp. Note that any filtering options will change what is displayed here. The visual timeline is a useful tool for seeing how much a beatmap has been modded at a glance. A tightly packed timeline would mean that a beatmap has received a lot of attention already.
 
-This displays all of the mods made for said difficulty that has a timestamp. Clicking on these will scroll down to the mod posted at that timestamp.
+The selected difficulty's difficulty settings are displayed on top of the visual timeline. From left-to-right these describe a difficulty's `Length`, `BPM`, `Circle Count` and `Slider Count`.
 
 ## Beatmap status
 
 ![](img/beatmap-status.jpg "Contains important information!")
 
-### Beatmap creator
-
-This contains the creator who submitted the beatmapset, shows the submission date, and the date the beatmap was last updated.
-
-### Watch/Unwatch
-
-This replaces beatmap subscription. You can track the beatmaps' status and open issues. This page is accessed via the website's Home page by clicking on [`modding watchlist` link](https://osu.ppy.sh/beatmapsets/watches) located near the top of the page.
-
-### Hype Train
-
-*Note: You now have to click `Hype` to add hype; giving praise will no longer add hype automatically.*
+The beatmap status bar shows everything relevant to a beatmap's spot in the [Beatmap Ranking Procedure](/wiki/Beatmap_ranking_procedure). This includes:
 
-Hype train is similar to the previous star priority system.
+- Hype train
+- General info
+- Watch/Unwatch buttons
+- Beatmap Page button
 
-> Adding praise will raise this beatmap's hype, making it more visible for nomination and ranking!
+### Hype train
 
-<!-- *Note: Players are required to play through a difficulty in the beatmapset in order to add Hype to a beatmap.* -->
+The Hype Train tracks how many [hypes](/wiki/Glossary#hype) a mapset has received. Once 5 hypes have been achieved, the beatmap can be nominated by [Beatmap Nominators](/wiki/Beatmap_Nominators). Hypes can only be given by going to the `General (All Difficulties)` tab. It will not work through the `General` or the `Timeline` tabs. Clicking the `Hype` button on the beatmap status bar will automatically move you to the correct tab.
```

Comment:
```suggestion
The hype train tracks how many [hypes](/wiki/Glossary#hype) a mapset has received. Once 5 hypes have been achieved, the beatmap can be nominated by [Beatmap Nominators](/wiki/Beatmap_Nominators). Hypes can only be given by going to the `General (All Difficulties)` tab. It will not work through the `General` or the `Timeline` tabs. Clicking the `Hype` button on the beatmap status bar will automatically move you to the correct tab.
```